packages/web-components/fast-foundation/src/skeleton/skeleton.template.ts

```diff
@@ -0,0 +1,13 @@
+import { html } from "@microsoft/fast-element";
+import { Skeleton } from "./skeleton";
+
+export const SkeletonTemplate = html<Skeleton>`
+  <template
+    aria-busy="${x => x.ariaBusy}"
+    shape="${x => x.shape}"
+    class="${x => (x.shape === "circle" ? "circle" : "rect")}"
```

Comment:
Rather than set the class on the template you could also manipulate the classlist directly in skeleton.ts.  For example this.classlist.toggle("myClass", this.showMyClass === true)

Reply:
@nicholasrice  are we still looking to do this in the template when possible?